kafka-ui-react-app/src/components/Schemas/Details/__test__/LatestVersionItem.spec.tsx

```diff
@@ -27,7 +37,11 @@ describe('LatestVersionItem', () => {
 
   it('matches snapshot', () => {
     expect(
-      shallow(<LatestVersionItem schema={jsonSchema} />)
+      shallow(
```

Comment:
Oleg told us that we have to replace the enzyme with a react-testing-library.

Reply:
@NelyDavtyan is right. But @Dekshut never mind for now it is enough